app/views/categories/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="text-center mt-4"> Categories </h1>
+
+<div class="flickr_pagination">
+  <%= will_paginate @categories, :container => false %>
+</div>
+
+<div class="container">
+<% @categories.each do |category| %>
+  <div class="row justify-content-md-center">
+    <div class="col-6 mt-4">
+      <div class="card text-center shadow mb-4 bg-white rounded">
+        <div class="card-body">
+          <h3 class="card-title"> <%= link_to category.name, category_path(category), class: "text-info" %> </h3>
```

Comment:
Wen  Zeilen zu lang werden (> 100 Zeichen), gern in mehrere Zeilen aufteilen. Z.B.:
```
<h3 class="card-title"> 
  <%= link_to category.name, category_path(category), class: "text-info" %> 
</h3>
```

Reply:
Ich habe ein paar html.erb-Dateien entsprechend deinem Hinweis geändert. Könntest du bitte überprüfen, ob sie in Ordnung sind? 

https://github.com/aboutsource/alpha-blog/pull/4/commits/bcc642605b88e8d3a15d0d9e2319ad306591478d

Ich finde `_navigation.html.erb` besonders überwältigend, was die superlangen Zeilen angeht. Könntest du auch hier vorschlagen, wie ich auch hier die Zeilen aufteilen könnte?

https://github.com/aboutsource/alpha-blog/blob/main/app/views/layouts/_navigation.html.erb